NineChronicles.Headless/GraphTypes/States/WorldBossKillRewardRecordType.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using GraphQL.Types;
+using Nekoyume.Model.State;
+
+namespace NineChronicles.Headless.GraphTypes.States
+{
+    public class WorldBossKillRewardRecordType : ObjectGraphType<WorldBossKillRewardRecord>
+    {
+        private class WorldBossKillRewardRecordMapType : ObjectGraphType<KeyValuePair<int, bool>>
+        {
+            public WorldBossKillRewardRecordMapType()
+            {
+                Field<IntGraphType>("bossLevel", resolve: context => context.Source.Key);
+                Field<BooleanGraphType>(
+                    "claimed",
+                    description: "check reward already claimed. if already claimed return true.",
+                    resolve: context => context.Source.Value);
+            }
+        }
+
+        public WorldBossKillRewardRecordType()
+        {
+            Field<ListGraphType<WorldBossKillRewardRecordMapType>>(
+                "map",
+                resolve: context => context.Source);
```

Comment:
The map and those elements seems non-nullable. If there is no records, it will return empty list `{ map: [] }`.